tests/reading-tests.md

```diff
@@ -0,0 +1,397 @@
+# Reading Tests
```

Comment:
I don't see assets/tests_tests_replit-test-success.png used in this file (or any other in this PR). Can it be removed?